docs/get_started/install.md

```diff
@@ -1066,7 +1075,15 @@ To install python bindings run the following commands in the MXNet directory:
 
 ```bash
     cd python
-    sudo python setup.py install
+    pip install --upgrade pip
+    pip install -e .
```

Comment:
They are virtually the same but pip seems to be the preferred method as it has [several advantages](https://stackoverflow.com/questions/15724093/difference-between-python-setup-py-install-and-pip-install).

Generally it is more convenient for users (they can also `pip uninstall` the package, the -e option might be more known/convenient than python setup.py develop).